src/main/java/Duke/DeadLineCommand.java

```diff
@@ -0,0 +1,22 @@
+package Duke;
+import java.time.LocalDateTime;
+
+public class DeadLineCommand extends Command {
```

Comment:
You can improve the naming here. I am pretty sure this is Deadline, not Dead Line right?